Lib/timeSeries.py

```diff
@@ -24,21 +33,29 @@ def __init__(self):
         self.time = [] 
         self.pos = []
         self.sig = []
-        self.covar = []
+        self.corr = []
         self.frame = 'NO FRAME SET'
-        self.refPos = [0,0,0]
+        self.coordType = 'NO TYPE SET'
+        self.refPos = [0.0,0.0,0.0]
 
+    ####################################################################
     def readUnrTxyz2(self, fileName):
 
         """
-        Function to read in UNR-style .txyz2 file and assign 
-        values to timeSeries class object
+        Read in UNR-style .txyz2 file and assign values to timeSeries 
+        class object
```

Comment:
Might wanna add a note that you're expecting a particular filename format 'cause you pull `self.frame` out of it.

Reply:
UNR mandates the file naming convention. So by saying it reads in a UNR-style .txyz2 file the file naming should be implied. But maybe I could add:
"UNR file naming convention enforced."